roles/privacyidea/tasks/main.yml

```diff
@@ -1,3 +1,12 @@
 ---
+- import_tasks: install_packages.yml
+- import_tasks: setup_directories.yml
+- import_tasks: install_privacyidea.yml
+- import_tasks: setup_database.yml
+- import_tasks: generate_keys.yml
+- import_tasks: configure_privacyidea.yml
+- import_tasks: create_admin.yml
+- import_tasks: setup_nginx.yml
 - import_tasks: security.yml
-- import_tasks: installation.yml
+# Import Data is run only on spsh-dev and spsh-staging; for spshinfra-dev, no data is imported.
+- import_tasks: import_data.yml
```

Comment:
if this task is not for every stage and should be enabled, disabled on command, maybe adding a var to control the behaviour is useful?

Reply:
Import Data task will run when privacyidea_execute_data_import is true